app/views/home/partials/_impacts.html.erb

```diff
@@ -0,0 +1,26 @@
+<carousel 
+  id="impacts"
+  class="carousel--impacts"
+  >
+  <% data['collection'].each_with_index do |slide, index| %>
+    <carousel-slide :margin-size="0" :impacts="true" >
+      <div class="carousel__slide">
+        <div class="carousel-slide__content">
+
+          <%= image_tag(slide['icon'], class: "carousel-slide__icon") %>
+
+          <div class="carousel-slide__body">
+            <div class="carousel-slide__subtitle">
+              <span class="carousel-slide__impact">OUR IMPACT · </span>
```

Comment:
Can this move to translations and use styling to make uppercase?

Reply:
I've moved the translatable text over to the .yml file and added a text-transform property to the element so it displays the text in upper case.